roles/hammerdb/templates/db_creation.yml

```diff
@@ -16,7 +16,8 @@ spec:
       - name: hammerdb
         command: ["/bin/sh", "-c"]
         args: ["/usr/local/bin/uid_entrypoint; cd /hammer; ./hammerdbcli auto /creator/createdb.tcl"]
-        image: "quay.io/cloud-bulldozer/hammerdb:latest"
+        #image: "quay.io/cloud-bulldozer/hammerdb:latest"
+        image: "quay.io/mkarg/hammerdb:latest"
```

Comment:
I assume snafu needs an update and then we can point this back to the cloud-bulldozer branch?

Reply:
We need to update snafu first, that's right.

Reply:
snafu is updated so should be safe to mve back to original branch>?

Reply:
see below - some idiot made a stupid mistake so we need another PR for snafu.